pending/potplayer.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2018 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-20
+
+-->
+<cleaner id="potplayer" os="windows">
+  <label>PotPlayer</label>
+  <description>Media Player</description>
+  <option id="rememberfiles">
+    <label>Remember Files</label>
+    <description>Delete the Remember Files list</description>
+        <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini\RememberFiles\*" />
+        <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini64\RememberFiles\*" />
+  </option>
+  <option id="logs">
+    <label>Log Files</label>
+    <description>Delete the Log Files</description>
+        <action command="delete" search="file" path="$APPDATA\Daum\PotPlayer\Log\*"/>
```

Comment:
```suggestion
        <action command="delete" search="glob" path="$APPDATA\Daum\PotPlayer\Log\*"/>
```

Reply:
What's about `search="walk.files"`?

Reply:
If you want to recursively delete all files under the directory `Log`, yes: use walk.files and take off the wildcard.

Reply:
Yes, did that!